examples/profile/main.go

```diff
@@ -61,7 +85,13 @@ func profile(w http.ResponseWriter, r *http.Request) {
 
 	activityDetails, errStrings := client.GetActivityDetails(yotiOneTimeUseToken)
 	if len(errStrings) != 0 {
-		log.Fatalf("Errors: %v", errStrings)
+		errorPage(w, r.WithContext(context.WithValue(
+			r.Context(),
+			contextKey("yotiError"),
+			fmt.Sprintf("Errors: %v", errStrings),
```

Comment:
I know it's just using what was previously there, but I don't think we need the `Errors: ` here, since when the error is displayed, we already have `Error: `, so it's `Error: Errors: [ProfileNotFound]`. 

So if we remove "Errors" here, we'll just have `Error: [ProfileNotFound]`